srcpkgs/python3-pep517/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-pep517'
+pkgname=python3-pep517
+version=0.9.1
+revision=1
+wrksrc=pep517-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 python3-toml"
+short_desc="Wrappers to build Python packages using PEP 517 hooks"
+maintainer="Paper <paper@tilde.institute>"
+license="MIT"
+homepage="https://pep517.readthedocs.io/en/latest/"
+distfiles="${PYPI_SITE}/p/pep517/pep517-${version}.tar.gz"
+checksum=aeb78601f2d1aa461960b43add204cc7955667687fbcf9cdb5170f00556f117f
+
+do_check() {
+	: # tests need flit_core v2, but we have v3 packaged
```

Comment:
Rather than skipping with `true`, I think it is good practice to print a message explaining why the tests are skipped:
```
echo "skipping check: tests need flit_core v2, but only v3 is packaged"
```

Reply:
I know some templates do this, but it would be nice to settle on something... The concise solution is using `make_check=no` with a comment on top, but it won't print anything.

Reply:
`make_check=no` and dropping the custom `do_check` is OK too.